### PR TITLE
Merging template updates 1.14

### PR DESCRIPTION
**Pull request by d4straub**
This is an update to nf-core tools 1.14

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -74,6 +74,23 @@ workflow.onError {
     NfcoreSchema.validateParameters(params, json_schema, log)
 }
 
-////////////////////////////////////////////////////
-/* --                  THE END                 -- */
-////////////////////////////////////////////////////
+def checkHostname() {
```
Comment (skrakau) [inline main.nf:77]: I think this is only for DSL1 pipelines, there is already a check for hostnames in `lib/Checks.groovy` https://github.com/nf-core/ampliseq/blob/ffb2fa5d65bf33b4937a2f4c2f8d080776e0cba5/lib/Checks.groovy#L49

**Review by skrakau — CHANGES_REQUESTED** [review]
Looks good, just the question with the `checkHostname()` function.

File: main.nf (hunk 1 above)
Comment (d4straub) [reply main.nf:77]: Ah haven't seen, good catch, will test if it throws an error when I remove it!

**Review by skrakau — APPROVED** [review]

**Comment by d4straub** [comment]
Thanks!

**Pull request merged**